docs/contributing.md

```diff
@@ -0,0 +1,267 @@
+# Contributing to the monorepo
```

Comment:
The majority of these contributor docs holds true for the design system repo which is why I'm opting to keep it. There are some instnaces of `metamask-module-template` that need to be replaced once we have merged/shipped our first package. Issue here https://github.com/MetaMask/metamask-design-system/issues/27